.github/pull_request_template.md

```diff
@@ -7,30 +7,24 @@ https://github.com/lucyparsons/OpenOversight/blob/develop/CONTRIB.md#recommended
 Also make sure you have read and abide by the code of conduct:
 https://github.com/lucyparsons/OpenOversight/blob/develop/CODE_OF_CONDUCT.md
 
+If this pull request is not ready for review yet, please submit it as a draft.
 -->
-
-## Status
-
-Ready for review / in progress
+**Fixes issue:** (LINK YOUR ISSUE RIGHT HERE)
 
 ## Description of Changes
 
-Fixes # .
-
-Changes proposed in this pull request:
 
+## Changes proposed in this pull request:
  -
  -
  -
 
 ## Notes for Deployment
 
+
 ## Screenshots (if appropriate)
 
-## Tests and linting
 
+## Tests and linting
  - [ ] I have rebased my changes on current `develop`
-
- - [ ] pytests pass in the development environment on my local machine
-
- - [ ] `pre-commit` checks pass
```

Comment:
Since the `pre-commit` step is in the GitHub action, this checkbox is redundant. If it doesn't pass, you can't merge it anyways.

Reply:
Part of this was meant as a reminder, but I am fine with removing it

Reply:
I added a similar line back to it.